Replicator/IncomingRev.cc

```diff
@@ -260,19 +260,22 @@ namespace litecore { namespace repl {
 
 
     void IncomingRev::_revisionInserted() {
+        Retained<IncomingRev> retainSelf = this;
```

Comment:
Would it make more sense to move this self-retain into `finish()` itself?

Reply:
Maybe.  I'm not sure what scope it is trying to keep alive.

Reply:
I don’t remember the details anymore. It may be that the code changed some after I added the self-retain, and that revisionInserted used to call something else that was triggering deletion. With the code as it is, it must be the finish method that could delete, so the self-retain could go there too, but no problem leaving it where it is.